x/txfees/types/keys.go

```diff
@@ -11,16 +11,21 @@ const (
 	// RouterKey is the message route for slashing.
 	RouterKey = ModuleName
 
-	// FeeCollectorName the module account name for the fee collector account address.
-	FeeCollectorName = "fee_collector"
+	// NonNativeTxFeeCollectorName is the module account name for the alt fee collector account address (used for auto-swapping non-OSMO tx fees).
+	NonNativeTxFeeCollectorName = "non_native_fee_collector"
```

Comment:
SLightly worried that this used to be `FeeCollectorForStakingRewardsName` and what happens if there are leftover funds in the addr

Reply:
Not sure I am tracking the point, @ValarDragon the address is exactly the same, I just changed the reference name of it because it was confusing.

Reply:
great job on comments inthis file